### PR TITLE
User modal

### PR DESCRIPTION
**Pull request by marsh37**
This makes the user modal.

Use these SQL queries to test:

INSERT INTO users (fName, lName, email, phoneNum, createdAt, updatedAt) Values ("VALUE", "TEST", "george.hartman@eagles.oc.edu", "4056979595", 1, 1);
INSERT INTO users (fName, lName, email, phoneNum, createdAt, updatedAt) Values ("fName2", "lName2", "test1.edu", "4056979595", 1, 1);
INSERT INTO users (fName, lName, email, phoneNum, createdAt, updatedAt) Values ("fName3", "lName3", "test2.edu", "4056979595", 1, 1);
INSERT INTO users (fName, lName, email, phoneNum, createdAt, updatedAt) Values ("fName4", "lName4", "test3.edu", "4056979595", 1, 1);
INSERT INTO users (fName, lName, email, phoneNum, createdAt, updatedAt) Values ("fName5", "lName5", "test4.edu", "4056979595", 1, 1);

INSERT INTO roles (type, createdAt, updatedAt) VALUES ("admin", NOW(), NOW());

INSERT INTO userroles (createdAt, updatedAt, userId, roleId) VALUES (NOW(), NOW(), 1, 1);
INSERT INTO userroles (createdAt, updatedAt, userId, roleId) VALUES (NOW(), NOW(), 3, 1);
INSERT INTO userroles (createdAt, updatedAt, userId, roleId) VALUES (NOW(), NOW(), 5, 1);

### CODE REVIEW

File: app/controllers/auth.controller.js [hunk 1]
```diff
@@ -59,14 +62,31 @@ exports.login = async (req, res) => {
...
+          }
+        ],
+      },
+    ]
+
   })
     .then((data) => {
       if (data != null) {
         user = data.dataValues;
+        console.log("THIS IS THE THING:", user)
```
Comment (KyleD0711) [inline app/controllers/auth.controller.js]: Probably get rid of this

File: app/controllers/auth.controller.js [hunk 2]
```diff
@@ -77,6 +97,8 @@ exports.login = async (req, res) => {
       }
     })
     .catch((err) => {
+      console.log("THIS IS THE ERROR:", err)
```
Comment (KyleD0711) [inline app/controllers/auth.controller.js]: Probably get rid of this

File: app/controllers/userrole.controller.js [hunk 3]
```diff
@@ -2,25 +2,38 @@ const db = require("../models");
...
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message:
-          err.message || "Some error occurred while retrieving user roles.",
-      });
-    });
+
+  try {
+    const userRoles = await db.sequelize.query(
```
Comment (KyleD0711) [inline app/controllers/userrole.controller.js]: We need to talk about this

Comment (marsh37) [reply app/controllers/userrole.controller.js]: This sounds like a disappointed dad. I'm scared.

Comment (KyleD0711) [reply app/controllers/userrole.controller.js]: Sorry to freak you out lol

File: app/controllers/userrole.controller.js [hunk 4]
```diff
@@ -2,25 +2,67 @@ const db = require("../models");
...
+      id: userId,
+    },
+    raw: true,
+  });
+
+
+  
+  try {
+    const userRoles = await db.sequelize.query(
+      `SELECT
```
Comment (KyleD0711) [inline app/controllers/userrole.controller.js]: Just need to get rid of this

**Pull request merged**